docs/src/manual/input.rst

```diff
@@ -18,63 +18,356 @@ that can help!
 Tools able to create chemiscope input
 -------------------------------------
 
-``chemiscope`` Python module
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Note that chemiscope does not compute structural representations or
+dimensionality reduction, and the command line interface works iff
+there are mappable quantities in the file. You can generate such representations
+or reductions with packages such as `ASAP`_ or `scikit-matter`_.
+The `ASAP`_ structural analysis package is another tool that can directly
+generate an output in chemiscope format.
 
-The easiest way to create a JSON input file is to use the ``chemiscope`` Python
-module. Install the package with ``pip install chemiscope``, and use
+The easiest way to create a JSON input file is to use the ``chemiscope`` 
+:ref:`Python module <python-module>`. 
+Install the package with ``pip install chemiscope``, and use
 :py:func:`chemiscope.write_input` or :py:func:`chemiscope.create_input` in your
 own script to generate the JSON file.
 
 If all the properties you want to include into chemiscope are already stored in
 a file `ase`_ can read, the ``chemiscope`` python package also installs a
-`chemiscope-input <chemiscope-input-cli_>`_ command line script.
+:ref:`chemiscope-input <chemiscope-input-cli>` command line script.
 
-Note that chemiscope does not compute structural representations or
-dimensionality reduction, and the command line interface works iff
-there are mappable quantities in the file. You can generate such representations
-or reductions with packages such as `ASAP`_ or
-`scikit-matter`_.
-The `ASAP`_ structural analysis package is another tool that can directly
-generate an output in chemiscope format.
 
+Input file reference
+--------------------
 
-``chemiscope`` functions reference
-----------------------------------
+If you can not or do not want to use the ``chemiscope`` python package to create
```

Comment:
Can we add a snarky tooltip here?

Reply:
Yes we can. Please do it!

Reply:
@rosecers this looks relevant: https://sphinx-hoverxref.readthedocs.io/en/latest/usage.html